pages.hi/common/$.md

```diff
@@ -0,0 +1,24 @@
+# Dollar sign
+
+> बैश वैरिएबल का विस्तार करें।
+> अधिक जानकारी: <https://gnu.org/software/bash/manual/bash.html#Shell-Variables>.
```

Comment:
```suggestion
> अधिक जानकारी: <https://gnu.org/software/bash/manual/bash.html#Shell-Variables>|
```
Sorry for the inconvenience, | is the period we use across all places in Hindi pages.